server/api/operator.go

```diff
@@ -280,11 +280,35 @@ func (h *operatorHandler) Post(w http.ResponseWriter, r *http.Request) {
 			h.r.JSON(w, http.StatusInternalServerError, err.Error())
 			return
 		}
+	case "scatter-regions":
+		// support both receiving key ranges or regionIDs
+		startKey, _ := input["start_key"].(string)
+		endKey, _ := input["end_key"].(string)
+		regionIDs, _ := input["region_ids"].([]uint64)
```

Comment:
Does the client need to split region first?

Reply:
scatter and split should be 2 different API.

Reply:
ok